integration_tests/integration_tests/tests/test_volume_anomalies.py

```diff
@@ -10,38 +11,82 @@
 
 def test_anomalyless_table_volume_anomalies(test_id: str, dbt_project: DbtProject):
     data = [
-        {TIMESTAMP_COLUMN: date.strftime(DATE_FORMAT)}
-        for date in generate_dates(base_date=date.today())
+        {TIMESTAMP_COLUMN: cur_date.strftime(DATE_FORMAT)}
+        for cur_date in generate_dates(base_date=date.today())
     ]
     test_result = dbt_project.test(data, test_id, DBT_TEST_NAME, DBT_TEST_ARGS)
     assert test_result["status"] == "pass"
 
 
 def test_full_drop_table_volume_anomalies(test_id: str, dbt_project: DbtProject):
     data = [
-        {TIMESTAMP_COLUMN: date.strftime(DATE_FORMAT)}
-        for date in generate_dates(base_date=date.today())
-        if date < date.today() - timedelta(days=1)
+        {TIMESTAMP_COLUMN: cur_date.strftime(DATE_FORMAT)}
+        for cur_date in generate_dates(base_date=date.today())
+        if cur_date < cur_date.today() - timedelta(days=1)
     ]
     test_result = dbt_project.test(data, test_id, DBT_TEST_NAME, DBT_TEST_ARGS)
     assert test_result["status"] == "fail"
 
 
 def test_partial_drop_table_volume_anomalies(test_id: str, dbt_project: DbtProject):
     data = [
-        {TIMESTAMP_COLUMN: date.strftime(DATE_FORMAT)}
-        for date in generate_dates(base_date=date.today())
-        for _ in range(2 if date < date.today() - timedelta(days=1) else 1)
+        {TIMESTAMP_COLUMN: cur_date.strftime(DATE_FORMAT)}
+        for cur_date in generate_dates(base_date=date.today())
+        for _ in range(2 if cur_date < cur_date.today() - timedelta(days=1) else 1)
     ]
     test_result = dbt_project.test(data, test_id, DBT_TEST_NAME, DBT_TEST_ARGS)
     assert test_result["status"] == "fail"
 
 
 def test_spike_table_volume_anomalies(test_id: str, dbt_project: DbtProject):
     data = [
-        {TIMESTAMP_COLUMN: date.strftime(DATE_FORMAT)}
-        for date in generate_dates(base_date=date.today())
-        for _ in range(1 if date < date.today() - timedelta(days=1) else 2)
+        {TIMESTAMP_COLUMN: cur_date.strftime(DATE_FORMAT)}
+        for cur_date in generate_dates(base_date=date.today())
+        for _ in range(1 if cur_date < cur_date.today() - timedelta(days=1) else 2)
     ]
     test_result = dbt_project.test(data, test_id, DBT_TEST_NAME, DBT_TEST_ARGS)
     assert test_result["status"] == "fail"
+
+
+@Parametrization.autodetect_parameters()
+@Parametrization.case(name="source", as_model=False)
+@Parametrization.case(name="model", as_model=True)
```

Comment:
Does the `name` do something?

Reply:
it affects the test id
will be `test_volume_anomalies_with_where_parameter[model]` or `test_volume_anomalies_with_where_parameter[source]`